examples/usvg_viewer/src/asset.rs

```diff
@@ -0,0 +1,73 @@
+use anyhow::{bail, Result};
+use generic_array::GenericArray;
+use hex_literal::hex;
+use sha2::{Digest, Sha256};
+use std::env::temp_dir;
+use std::io::Read;
+use std::path::PathBuf;
+
+pub struct SvgAsset {
+    url: &'static str,
+    sha256sum: [u8; 32],
+    pub license: &'static str,
+    pub size: u128,
+}
+
+impl SvgAsset {
+    pub fn local_path(&self) -> PathBuf {
+        let arr = GenericArray::from(self.sha256sum);
```

Comment:
I'm not sure exactly what you're suggesting here. The `LowerHex` implementation for `GenericArray` is around 30 lines of code. Do you want to inline that? Or do you know of a different way of printing `[u8; 32]` to hex?

I'm using `GenericArray` is because that's what `Sha256::digest` returns. It'll be in the dependency graph whether I use it for hex-printing or not.

Reply:
It's unfortunate that there's doesn't seem to be a minimal sha256 crate which just gives us a `[u8; 32]`.